examples/contracts/multicall/index.ts

```diff
@@ -1,42 +1,42 @@
-import { createPublicClient, http } from "viem";
-import { mainnet } from "viem/chains";
-import { wagmiContract } from "./contract";
+import { createPublicClient, http } from 'viem'
+import { mainnet } from 'viem/chains'
+import { wagmiContract } from './contract.js'
```

Comment:
Drop the .js extensions for examples?

```suggestion
import { wagmiContract } from './contract'
```

Reply:
Ok, fair point. Let's just do `src` for now (I think there's a good chance that `nodenext` will catch on and hopefully eventually also reach applications, it has really grown on me tbh, but until then... indeed let's jsut do ./src)